scripts/run_test_chunk.sh

```diff
@@ -12,4 +12,10 @@ else
   TAGS="${TAGS} and not @ExcludeOnLegacyUI"
 fi
 
+if [ "${PHASE2_CORE_CANARY_RATIO}x" == "1.0x" ] || [ "${PHASE2_CORE_CANARY_RATIO}x" == "1x" ]; then
```

Comment:
Why do we add "x" at the end? 🤔

Reply:
It's to avoid empty strings which potentially result in syntax errors or unintended behaviour depending on the shell tool:

```
if [ "" == "1.0" ]
```

vs

```
if [ "x" == "1.0x" ]
```